spec/models/piece_spec.rb

```diff
@@ -1,5 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Piece, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:game) { create(:game) }
+  describe "instantiation" do
+    it "sets the white queen image correctly" do
+      expect(game.queens.where(color: true).first.image_name)
+        .to eq('white-queen.png')
+    end
+  end
```

Comment:
add test to make sure image is set correctly. only checking one piece for now, maybe throw a black piece test in there too later on?